src/core/event-handler.js

```diff
@@ -58,22 +58,28 @@ class EventHandler {
      * the callback is limited to 8 arguments.
      * @param {object} [scope] - Object to use as 'this' when the event is fired, defaults to
      * current this.
-     * @param {boolean} [once=false] - If true, the callback will be unbound after being fired once.
+     * @param {boolean} [once] - If true, the callback will be unbound after being fired once.
```

Comment:
`scope` and `once` should not be marked as optional (square brackets) here because both `on` and `once` do supply these parameters in all circumstances.